aws/resource_aws_redshift_cluster_test.go

```diff
@@ -140,6 +117,16 @@ func TestAccAWSRedshiftCluster_basic(t *testing.T) {
 					resource.TestMatchResourceAttr("aws_redshift_cluster.default", "dns_name", regexp.MustCompile(fmt.Sprintf("^tf-redshift-cluster-%d.*\\.redshift\\..*", ri))),
 				),
 			},
+			{
+				ResourceName:      "aws_redshift_cluster.default",
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{
+					"enable_logging",
```

Comment:
Does it make sense to ignore removed attributes or should we be checking that the new attribute is set on import?